app/models/user.rb

```diff
@@ -2,7 +2,9 @@ class User < ApplicationRecord
   has_many :posts, class_name: 'Post'
   has_many :comments, class_name: 'Comment'
   has_many :likes, class_name: 'Like'
-
+  # validations
+  validates :Name, presence: true
```

Comment:
Optional 
- [ ] I can see you create a column with a capital letter. To best practice, you can use a small letter.
Name, Photo, Bio etc..